packages/node-cli/src/ApplicationConstants.js

```diff
@@ -6,6 +6,7 @@
 
 module.exports = {
 	SDK_COMMANDS_METADATA_FILE: 'metadata/SdkCommandsMetadata.json',
+	SDK_COMMANDS_METADATA_PATCH_FILE: 'metadata/SdkCommandsMetadataPatch.json',
```

Comment:
I think it would be nice to put a comment here explaining what this file does:
The file that overwrites selected properties in SdkCommandsMetadata.json file that are specific for SuiteCloud CLI for Node.js